src/locale/locales/pt-BR/messages.po

```diff
@@ -4340,28 +4340,28 @@ msgstr "Nota: o Bluesky é uma rede aberta e pública. Esta configuração limit
 
 #: src/screens/Messages/List/index.tsx:215
 msgid "Nothing here"
-msgstr ""
+msgstr "Não há nada aqui"
 
 #: src/view/screens/NotificationsSettings.tsx:54
 msgid "Notification filters"
-msgstr ""
+msgstr "Filtros de Notificação"
```

Comment:
"Notificação" should not be capitalized here

Reply:
ok